bundlewrap/utils/ui.py

```diff
@@ -217,6 +217,14 @@ def __init__(self):
         self._status_line_present = False
         self._waiting_for_input = False
 
+    def __enter__(self):
+        self.activate()
+        # return self, so users could also do `with IOManager() as io:`
+        return self
+
+    def __exit__(self):
```

Comment:
https://docs.python.org/3/reference/compound_stmts.html#with says this needs more arguments 🤔